doc/specs/vulkan/chapters/VK_EXT_debug_report.txt

```diff
@@ -223,14 +231,11 @@ include::../api/protos/vkDebugReportMessageEXT.txt[]
   * pname:objectType is a elink:VkDebugReportObjectTypeEXT specifying the
```

Comment:
These still feel copy-pasted and not applicable to `vkDebugReportMessageEXT`.
Should be something like:

> * `instance` is the debug stream's `VkInstance`.
> * `flags` indicates the `VkDebugReportFlagBitsEXT` classification of this event/message.